java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -730,18 +731,18 @@ class ReferenceModeStore private constructor(
                 StoreOptions(
                     storageKey = storageKey.storageKey,
                     type = refType,
-                    versionToken = options.versionToken,
-                    coroutineContext = options.coroutineContext
+                    versionToken = options.versionToken
                 )
             )
 
             return ReferenceModeStore(
                 refableOptions,
                 containerStore,
                 storageKey.backingKey,
-                options.coroutineContext,
                 type.containedType
-            )
+            ).also { store ->
+                it.initiateCleanupFlow()
+            }
```

Comment:
You've got `store` and `it` both here... is this intended?

Reply:
Nope, mistake.